webapp/channels/src/components/admin_console/set_by_env.tsx

```diff
@@ -4,7 +4,7 @@
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
 
-export default function SetByEnv() {
+const SetByEnv: React.FunctionComponent = (): JSX.Element => {
```

Comment:
we actually don't have to convert this to an anonymous function. its fine as it was before with default export. This conversion might actually be changing the file extension :D